source/index.html

```diff
@@ -1,12 +1,159 @@
+      <h2 class="title">Функциональное питание для котов</h2>
+      <p class="announcement__slogan">
+        Занялся собой? Займись котом!
+      </p>
+      <a class="announcement__selection" href="#">Подобрать программу</a>
+      <img class="announcement__can" src="img/index-can-mobile.png" alt="index-can-mobile" width="280" height="270">
+    </section>
+    <section class="diet">
+      <div class="diet__wrapper">
+        <img class="diet_cat-back" src="img/cat-back/cat_back-1.svg" alt="cat-back-1" width="50" height="50">
+        <h2 class="diet__title">Похудение</h2>
+        <p class="diet__intro">Ваш кот весит больше собаки и почти утратил способность лазить по деревьям? Пора на
+          диету! Cat Energy Slim
+          поможет вашему питомцу сбросить лишний вес.
+        </p>
+        <button class="diet__button">каталог slim</button>
+      </div>
+      <div class="diet__wrapper">
+        <img class="diet_cat-back" src="img/cat-back/cat_back-2.svg" alt="cat-back-2" width="50" height="50">
+        <h2 class="diet__title">Набор массы</h2>
+        <p class="diet__intro">Заработать авторитет среди дворовых котов и даже собак? Серия Cat Energy Pro
+          поможет вашему коту нарастить
+          необходимые мышцы!
+        </p>
+        <button class="diet__button">каталог pro</button>
+      </div>
+    </section>
+    <section class="advantages">
+      <div class="advantages__wrapper">
+        <h2 class="visually-hidden">Наши преимущества</h2>
+        <h3 class="advantages__title">Как это работает</h3>
+        <ul class="advantages__list">
+          <li class="advantages__item advantages__item--useful">
+            <p class="advantages__description">Функциональное питание содержит только полезные питательные вещества.</p>
+          </li>
+          <li class="advantages__item advantages__item--modern">
+            <p class="advantages__description">Выпускается в виде порошка, который нужно лишь залить кипятком и готово.
+            </p>
+          </li>
+          <li class="advantages__item advantages__item--functional">
+            <p class="advantages__description">Замените один-два приема обычной еды на наше функциональное питание.</p>
+          </li>
+          <li class="advantages__item advantages__item--result">
+            <p class="advantages__description">Уже через месяц наслаждайтесь изменениями к лучшему вашего питомца!</p>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="example">
+      <div class="example__wrapper">
+        <h2 class="example__title"></h2>
+        <p class="example__description">Борис сбросил 5 кг за 2 месяца, просто заменив свой обычный корм на Cat Energy
+          Slim. Отличный результат без изнуряющих тренировок! При этом он не менял своих привычек и по-прежнему спит по
+          16 часов в день.</p>
+        <dl class="example__list">
+          <dt class="example__number">
+            5 кг</dt>
```

Comment:
Для наглядности лучше закрывающий перенести
```suggestion
          <dt class="example__number">
            5 кг
          </dt>
```